packages/material-ui/package.json

```diff
@@ -11,7 +11,8 @@
   "scripts": {
     "start": "tsdx watch",
     "build": "rimraf dist && tsdx build",
-    "test": "tsdx test"
+    "test": "tsdx test",
+    "updateSnapshot": "tsdx test -u"
```

Comment:
You can probably revert both of the `package*.json` files since they really don't affect the rest of this change and the `package-lock.json` change was built using node 16 which we don't yet support.